tools/tcptracer_example.txt

```diff
@@ -0,0 +1,25 @@
+Demonstrations of tcptracer, the Linux eBPF/bcc version.
+
+
+This tool traces the kernel function performing TCP connections (eg, via a
+connect() or accept() syscalls) and closing them (explicitly or if the process
+dies). Some example output (IP addresses are fake):
+
+```
+# ./tcptracer
+T  PID    COMM             IP SADDR            DADDR            SPORT  DPORT
+C 2389    telnet           4  192.168.1.2      192.168.1.1      53404  23
+C 23323   curl             6  [::1]            [::1]            36226  80
+X 2389    telnet           4  192.168.1.2      192.168.1.1      53404  23
+A 23278   nc               6  [::1]            [::1]            80     36226
+X 23323   curl             6  [::1]            [::1]            36226  80
+X 23278   nc               6  [::1]            [::1]            80     36226
+A 15195   nc               4  10.202.210.1     10.202.109.12    8080   43904
+X 15195   nc               4  10.202.210.1     10.202.109.12    8080   43904
```

Comment:
Ooops. No, that was me being off-by-one when editing the file. Let me re-run the test scenario.